mapdata/WarcraftLegacies/UnitData/Core/edtm.json

```diff
@@ -63,7 +63,7 @@
     {
       "Id": 1731289461,
       "Type": 3,
-      "Value": "air,debris,ground,item,nonancient,structure,ward"
+      "Value": "debris,ground,item,nonancient,structure,ward"
```

Comment:
What's this change?

Reply:
2nd attack type for Druid of the talon (Storm crow form) not really sure how to fix this cause both attacks were basically identical before i removed air but alas it did nothing

Reply:
Cool, let's just un-attach the Druid of the Talon ticket from this PR then and it get merged in.